### PR TITLE
Ding dong

### PR DESCRIPTION
**Pull request by BDokas**

### CODE REVIEW

File: src/bot.py [hunk 1]
```diff
@@ -29,6 +29,10 @@ async def ping(ctx):
 async def bing(ctx):
     await ctx.send("bong")
 
+@bot.command(name="ding")
+async def dong(ctx):
+    await ctx.send("8====D")
```
Comment (gobisa) [inline src/bot.py:34]: change this

**Review by gobisa — CHANGES_REQUESTED** [review]

File: src/bot.py (hunk 1 above)
Comment (BDokas) [reply src/bot.py:34]: I disagree

Comment (gobisa) [reply src/bot.py:34]: For consistency, the string being sent should "dong"

Comment (dannyreilman) [reply src/bot.py:34]: I agree with @gobisa

**Comment by gobisa** [comment]
we're not about this here

**Pull request closed without merging**